internal/providers/terraform/azure/testdata/signalr_service_test/signalr_service_test.tf

```diff
@@ -0,0 +1,35 @@
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_signalr_service" "example" {
+  for_each = toset(["Free_F1", "Standard_S1", "Premium_P1"])
+
+  name                = "tfex-signalr"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  sku {
+    name     = each.value
+    capacity = 1
+  }
+}
+
+resource "azurerm_signalr_service" "example_with_usage" {
+  for_each = toset(["Standard_S1", "Premium_P1"])
+
+  name                = "tfex-signalr"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  sku {
+    name     = each.value
+    capacity = 5
+  }
+}
```

Comment:
Let's fix this missing newline here. Otherwise, when the next person opens this file in the editor and saves it, it will be added, resolving in a git diff line.

Reply:
I hear you (though for whatever reason goland doesn't do that) but I feel like these kind of style issues should be addressed through code (i.e. `make fmt`) not code reviews.  There are over 100 .tf files without trailing newline and for whatever reason `terraform fmt` [doesn't want to enforce this](https://github.com/hashicorp/terraform/issues/23223) but I think it should be easy enough to right a `find + awk` that does it so I'll do that in a separate pr.